src/harness/sas_interface.py

```diff
@@ -519,5 +519,58 @@ def TriggerDailyActivitiesImmediatelyAndWaitUntilComplete(self):
     If the status is not changed within 2 hours it will throw an exception.
     """
     pass
+
+  @abc.abstractmethod
+  def TriggerCreateActivityDump(self):
+    """
+    Trigger SAS to create a full activity dump
+    """
+    pass
+
+  def InjectPeerSas(self, request):
+    """
+        Inject peer SAS
+    """
+    pass
+
+
+class HttpServerInterface(object):
```

Comment:
Does this fit better in a different file? It seems unrelated to the normal SAS interface.

Reply:
I already put the HeepServerInterface into different file. I created a new file called httpserver_interface.py

Reply:
Looks like it hasn't been removed from this file?